Example/Example Swift/AppCoordinator.swift

```diff
@@ -101,20 +107,21 @@ final class AppCoordinator: Coordinator {
     fileprivate func showScreenAPI(with pages: [GiniCapturePage]? = nil) {
         documentMetadata = Document.Metadata(branchId: documentMetadataBranchId,
                                              additionalHeaders: [documentMetadataAppFlowKey: "ScreenAPI"])
-        let screenAPICoordinator = ScreenAPICoordinator(configuration: giniConfiguration,
+        let screenAPICoordinator = ScreenAPICoordinator(configuration: giniConfiguration, returnAssistantConfig: returnAssistantConfiguration,
```

Comment:
Is there a way to merge the return assistant configuration and gini capture configuration in the same configuration class?